apps/explorer/lib/explorer/chain/block.ex

```diff
@@ -122,15 +134,15 @@ defmodule Explorer.Chain.Block do
 
   use Explorer.Schema
 
-  alias Explorer.Chain.{Block, Hash, Transaction, ExternalTransaction, Wei}
+  alias Explorer.Chain.{Block, Hash, Transaction, Wei}
```

Comment:
Yes, if I understand correctly currently we will determine external transaction by markers in transaction hash. So, looks like we no need keep external transaction separately.